docs/enterprise/kubernetes-guides/eks/cluster.md

```diff
@@ -135,11 +135,24 @@ nodes will only be on the private subnet.
   ```
   You should see a cluster of a single node, with the status `Ready`.
 
+* Check that the `gp2` storage class is present and default:
```

Comment:
Does it have to be gp2? Or is the important part that there has to be a default? If so, lead with the latter and state that you will typically see gp2 if you provision a fresh EKS cluster.

Reply:
It's the default in the EBS CSI driver, and it is the only storage class that it creates by itself

Reply:
In a future iteration of the docs will add a section on how to add more storage classes

Reply:
```
Feldera needs at least a default storage class to allocate persistent volume claims (PVCs). Users can also explicitly specify the storage class per pipeline directly in the `PipelineConfig`.

If you installed the EBS CSI driver as per the instructions above, you should see the gp2 storage class in your cluster already, marked as the default.

  ```bash
  $ kubectl get sc
  NAME            PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
  gp2 (default)   kubernetes.io/aws-ebs   Delete          WaitForFirstConsumer   false                  64m
  ```
To set any configured storage class to be the default:

  kubectl patch sc my-sc -p '{"metadata":{"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'

```

Reply:
I've updated the wording, slight changes are "directly in their configuration" and the second sentence "If it has not been automatically marked as default (visible next to its name), set it as such using:"